components/raftstore-v2/src/router/message.rs

```diff
@@ -187,6 +193,7 @@ impl fmt::Debug for PeerMsg {
 
 pub enum StoreMsg {
     RaftMessage(Box<RaftMessage>),
+    CreatePeer(Box<CreatePeer>),
```

Comment:
`SplitInit` can be sent directly without using `CreatePeer`. The message can be constructed when needed.

Reply:
Okay.